.templates/syncthing/service.yml

```diff
@@ -0,0 +1,19 @@
+  syncthing:
+    image: linuxserver/syncthing:latest
+    container_name: syncthing
+    hostname: syncthing #optional
+    
+    environment:
+      - PUID=1000
+      - PGID=1000
+      - TZ=Europe/Madrid
+    volumes:
```

Comment:
Could you add a default mapping for data volume mounts? Without these all data is lost when the container is updated/recreated?
```yml
       - ./volumes/syncthing/data1:/data1
       - ./volumes/syncthing/data2:/data2
```
At least this is what [volume mappings](https://docs.linuxserver.io/images/docker-syncthing#parameters) seem to indicate?
IOTstack usually splits config and data into separate mappings. Haven't tested where synced folders are actually created by default.

Reply:
Sorry, didn't test my proposal. See my latest comment for my new proposal for volume mappings and the HOME environment variable.